assets/src/js/Components/ArchivePicker/ArchivePickerList.js

```diff
@@ -131,9 +132,17 @@ export default function ArchivePickerList() {
                 role="presentation"
               />
               {wordpress_id && (
-                <div className="added-to-library">
-                  <span>{__('Added to Media Library', 'planet4-master-theme-backend')}</span>
-                </div>
+                <>
+                  {(currentBlockImageId === wordpress_id) ? (
```

Comment:
What do you think if you just create only one `added-to-media-library` and the implement the logic only to the text?